telegram_export/exporter.py

```diff
@@ -81,8 +81,20 @@ def __init__(self, client, config, dumper, loop):
                 await self.downloader.start(entity)
         else:
             # Neither blacklist nor whitelist - get all
+            dialog_to_dump = (self.dumper.config.get('DialogWhitelist') or '').replace(' ' ,'').split(',')
             for dialog in await self.client.get_dialogs(limit=None):
-                await self.downloader.start(dialog.entity)
+                if dialog.is_user:
+                    dialogType = 'user'
+                elif dialog.is_group:
+                    dialogType = 'group'
+                elif dialog.is_channel:
+                    dialogType = 'channel'
+                else:
+                    dialogType = 'unknown!'
+                if dialogType in dialog_to_dump:
```

Comment:
String comparison? It's better if you have something like `user_enabled`, and check if `dialog.user and dialog.user_enabled`, or something.